### PR TITLE
Update version to 17.8

### PR DESCRIPTION
**Pull request by drewnoakes**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/9164)

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "17.7",
+  "version": "17.8",
```
Comment (smitpatel) [inline version.json:3]: Can you also drop buildNumberOffset?

**Review by smitpatel — CHANGES_REQUESTED** [review]

File: version.json (hunk 1 above)
Comment (drewnoakes) [reply version.json:3]: Drop as in remove, or as in lower? Can you propose a suggestion on the PR here?

File: version.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   "buildNumberOffset": 100,
```
Comment (smitpatel) [inline version.json:4]:
```suggestion
```

File: version.json (hunk 1 above)
Comment (smitpatel) [reply version.json:3]: As drop as in remove, added a suggestion.

**Review by smitpatel — APPROVED** [review]

**Pull request merged**